src/Network/GRPC/MQTT/Client.hs

```diff
@@ -64,58 +106,46 @@ import Network.GRPC.MQTT.Wrapping
     unwrapServerStreamResponse,
     unwrapUnaryResponse,
   )
-import Network.MQTT.Client
-  ( MQTTClient,
-    MQTTException (MQTTException),
-    MessageCallback (SimpleCallback),
-    normalDisconnect,
-  )
-import Network.MQTT.Topic (Topic (unTopic), mkTopic, toFilter)
-import qualified Proto.Mqtt as Proto
-import Proto3.Suite
-  ( Enumerated (Enumerated),
-    HasDefault,
-    Message,
-    def,
-    toLazyByteString,
-  )
-import Turtle (sleep)
-import UnliftIO (MonadUnliftIO)
-import UnliftIO.Async (withAsync)
-import UnliftIO.Exception (handle, onException)
-import UnliftIO.STM
-  ( TChan,
-    newTChanIO,
-    writeTChan,
+
+import Proto.Mqtt
+  ( AuxControl (AuxControlAlive, AuxControlTerminate),
+    AuxControlMessage (AuxControlMessage),
+    RemoteError,
   )
-import UnliftIO.Timeout (timeout)
+
+---------------------------------------------------------------------------------
 
 -- | Client for making gRPC calls over MQTT
+--
+-- @since 1.0.0
```

Comment:
```suggestion
-- @since 0.1.0.0
```

... and everywhere else. Though I don't think we've done a release yet so these aren't really necessary, IMHO.